### DISCUSSION TITLE
LFE Discussions on Github

### DISCUSSION THREAD

**Discussion by oubiwann** [lfe/lfe#451] [Announcements]
## Important

We've enabled this feature by request of the community, but please note that most discussions take place on Slack. We'll do our best to answer questions here, but there's a good chance that detailed discussions will continue to happen on Slack (even if they start here). We're just not that big of a community to be able to handle all modes of communications easily 😢

**Comment by jotaemei** [comment]
These were my thoughts too. I’m curious though if more open conversation could interest new people to get involved.

**Comment by rvirding** [comment]
Is there no way we can get discussions here also sent to slack and vice-versa?

**Reply by prertik** [reply]
Sending notification of created discussion or new comments can be done in slack via the Github app.

**Reply by prertik** [reply]
But, I don't think it can set to a channel. Users must subscribe via GH app on slack.
